src/compiler/generate-query-tests.xsl

```diff
@@ -72,6 +72,12 @@
       </xsl:if>
       <xsl:text>;&#10;</xsl:text>
 
+      <xsl:if test="x:helper">
+         <xsl:text>&#x0A;</xsl:text>
+         <xsl:text>(: User-provided library module(s) :)&#x0A;</xsl:text>
```

Comment:
Lowercase "user-provided"? Please see my similar comment in generate-xspec-tests.xsl.

Reply:
Fixed in 52d188fc3ac805d09aec692f9065c1daaa359900